pom.xml

```diff
@@ -966,6 +973,7 @@
                     <plugin>
                         <groupId>io.helidon.build-tools</groupId>
                         <artifactId>helidon-enforcer-plugin</artifactId>
+                        <!-- TODO: should align to 3.4.0 here? -->
```

Comment:
Romain?

Reply:
I don't understand what you are asking. This is our own "enforcer" plugin and since it's housed in this repository we have to depend on a previously released version.

Reply:
oops, pushed